tests/Tests/Models/ComplexeHydration/Item.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\ComplexeHydration;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Table;
+
+/**
+ * @Entity
+ * @Table(name="item")
+ */
+#[Table(name: 'item')]
+#[Entity]
+class Item
+{
+    /**
+     * @var int
+     * @Column(type="integer")
+     * @Id
+     * @GeneratedValue
+     */
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private int|null $id;
+
+    /**
+     * @var string
+     * @Column(length=255)
+     */
+    #[ORM\Column(type: 'string', length: 255)]
+    private string|null $libelle;
+
+    /**
+     * @var Collection<int, CmsComment>
+     * @OneToMany(targetEntity="CmsComment", mappedBy="article")
+     * @JoinColumn(name="itemSeries", referencedColumnName="item_id")
+     */
+    #[ORM\OneToMany(targetEntity: ItemSerie::class, mappedBy: 'item', cascade: ['persist'], orphanRemoval: true)]
+    #[ORM\JoinColumn(name: 'itemSeries', referencedColumnName: 'item_id')]
+    private $itemSeries;
+
+    public function __construct()
+    {
+        $this->itemSeries = new ArrayCollection();
+    }
+
+    public function getId(): int|null
+    {
+        return $this->id;
+    }
+
+    public function getLibelle(): string|null
+    {
+        return $this->libelle;
+    }
+
+    public function setLibelle(string $libelle): self
+    {
+        $this->libelle = $libelle;
+
+        return $this;
+    }
+
+    /** @return Collection<int, ItemSerie> */
+    public function getItemSeries(): Collection
+    {
+        return $this->itemSeries;
+    }
+
+    public function addItemSeries(ItemSerie $itemSeries): self
```

Comment:
Please remove any unused getter, setter, adder and remover. Likewise, if a relationship doesn't need to be bidirectional for the bug to appear, please make it unidirectional